.github/workflows/build-phar-release.yml

```
@@ -1,61 +1,61 @@
-      - name: Backward compatibility
-        run: |
-          cp ${{ env.BIN_PHAR }} .build/phar/context-generator.phar
-
-      - name: 📤 Upload release assets
-        uses: softprops/action-gh-release@v2.2.1
-        if: startsWith(github.ref, 'refs/tags/')
-        with:
-          token: "${{ secrets.RELEASE_TOKEN }}"
-          files: |
-            ${{ env.BIN_PHAR }}
+---
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)